tests/scripts/all.sh

```diff
@@ -1005,8 +1029,6 @@ component_test_m32_o1 () {
     # Build again with -O1, to compile in the i386 specific inline assembly
     msg "build: i386, make, gcc -O1 (ASan build)" # ~ 30s
     scripts/config.pl full
-    scripts/config.pl unset MBEDTLS_MEMORY_BACKTRACE
-    scripts/config.pl unset MBEDTLS_MEMORY_BUFFER_ALLOC_C
     scripts/config.pl unset MBEDTLS_MEMORY_DEBUG
```

Comment:
`MBEDTLS_MEMORY_DEBUG` only has an effect with `MBEDTLS_MEMORY_BUFFER_ALLOC_C`, so there's no point in disabling it here.

Reply:
If we add a check for the dependency of `MBEDTLS_MEMORY_DEBUG` on `MBEDTLS_MEMORY_BUFFER_ALLOC_C` to `check_config.h` (as you suggest below), this must be kept.

Reply:
Ok, no, if we also exclude it from `config.pl full`, then we don't need this line.

Reply:
Addressed in 1263707.